cli/internal/action/exportDocumentation.go

```diff
@@ -0,0 +1,266 @@
+	}
+
+	// Open Documentation database
+	docDB, err := sqlite.InitInput(args.Documentation)
+	if err != nil {
+		slog.Debug("action.ExportDocumentation could not initialize documentation db", "documentation", args.Documentation, "error", err)
+		return err
+	}
+
+	// Get documentation
+	filters := &config.CheckDocumentation{}
+	if len(includes) == 0 {
+		filters.Include = append(filters.Include, config.DocumentationFilter{
+			Source:   "*",
+			Document: "**/*",
+		})
+	} else {
+		for _, include := range includes {
+			filters.Include = append(filters.Include, config.DocumentationFilter{
+				Source:   include.SourceID,
+				Document: include.DocumentPath,
+				Tags:     include.Tags.ToDocumentationFilterTags(),
+			})
+		}
+	}
+	for _, exclude := range excludes {
+		filters.Exclude = append(filters.Exclude, config.DocumentationFilter{
+			Source:   exclude.SourceID,
+			Document: exclude.DocumentPath,
+			Tags:     exclude.Tags.ToDocumentationFilterTags(),
+		})
+	}
+	documents, err := docs.GetFilteredDocs(filters, docDB)
+	if err != nil {
+		slog.Debug("action.ExportDocumentation could not get filtered documents", "error", err)
+		return err
+	}
+	slog.Info("Retrieved filtered documents", "documents", len(documents))
+
+	// Get sources
+	sources, err := docDB.GetAllSources(context.Background())
+	if err != nil {
+		slog.Debug("action.ExportDocumentation could not get sources", "error", err)
+		return err
+	}
+	sourcesMap := make(map[string]*sqlite.SOURCE)
+	for _, source := range sources {
+		sourcesMap[source.ID] = &source
+	}
+
+	// Output documentation
+	err = exportFs(documents, sourcesMap, args.Includes, args.Excludes, args.Documentation, outputAbsPath)
+	if err != nil {
+		slog.Debug("action.ExportDocumentation could not export to path", "error", err)
+		return err
+	}
+
+	slog.Info("Export Complete")
+
+	return nil
+}
+
+func exportFs(documents []*docs.FilteredDoc, sources map[string]*sqlite.SOURCE, includes []string, excludes []string, inputPath string, outputPath string) (err error) {
+	slog.Info(fmt.Sprintf("Exporting %d documents to %s", len(documents), outputPath))
+
+	// Create path
+	err = os.MkdirAll(outputPath, 0755)
+	if err != nil {
+		slog.Debug("action.exportFs could not MkdirAll for outputPath", "outputPath", outputPath, "error", err)
+		return
+	}
+
+	// Record the number of documents for each source for our README
+	sourcesCount := make(map[string]int)
+
+	// Output documents
+	for _, document := range documents {
+		count := sourcesCount[document.Document.SourceID]
+		sourcesCount[document.Document.SourceID] = count + 1
+
+		// Get dir and filename
+		dir := path.Join(outputPath, document.Document.SourceID, filepath.Dir(document.Document.ID))
+		var filename string
+		if strings.HasSuffix(document.Document.ID, "/") {
+			filename = "index.md"
+		} else {
+			filename = filepath.Base(document.Document.ID)
+		}
+		if !strings.HasSuffix(filename, ".md") {
+			filename = filename + ".md"
+		}
```

Comment:
nit: Consider a small simplification
```suggestion
		} else {
			filename = filepath.Base(document.Document.ID) + ".md"
		}
```